### PR TITLE
Faster setting for Platooning model

### PR DESCRIPTION
**Pull request by schillic**
This requires <strike> https://github.com/JuliaReach/Reachability.jl/pull/489 and</strike> https://github.com/JuliaReach/LazySets.jl/issues/1144 to work.

### CODE REVIEW

**Comment by schillic** [comment]
Without this setting, it currently takes 16 seconds to overapproximate the reach tube (500 sets) to hyperrectangles (why?!?), which we have to do four times. With this setting, the whole analysis takes 0.8 seconds.

**Comment by schillic** [comment]
Since https://github.com/JuliaReach/LazySets.jl/issues/1144 is delicate, for now I decided to bloat the guards a tiny bit.

File: models/ARCH/AFF/Platooning/Platooning.jl [hunk 1]
```diff
@@ -34,6 +34,7 @@ function platooning(;
     c1 = c2 = 5.   # clock constraints
     tb = 10.       # lower bound for loss of communication
     tc = tr = 20.  # upper bound for loss of communication (tc) and reset time (tr)
+    guard_bloating = 1e-10  # additional bloating to get a non-flat guard intersection
```
Comment (mforets) [inline models/ARCH/AFF/Platooning/Platooning.jl]: This is an issue related to floating point arithmetic. Perhaps let `guard_bloating = sqrt(eps(Float64))` which corresponds to the default relative tolerance in floating point.

Comment (schillic) [reply models/ARCH/AFF/Platooning/Platooning.jl]: Yeah, I hesitate to add this to the model at all. Ideally we would detect this internally, but it's difficult. Let me try with that number...

Comment (schillic) [reply models/ARCH/AFF/Platooning/Platooning.jl]: It worked :+1:

**Pull request merged**